testing/kuttl/e2e/exporter/10--tls-cluster.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster-cert
+type: Opaque
+data:
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJnakNDQVNpZ0F3SUJBZ0lSQU0vVDF1MXllNHZ3ek1SWEt1NGlWZVF3Q2dZSUtvWkl6ajBFQXdNd0h6RWQKTUJzR0ExVUVBeE1VY0c5emRHZHlaWE10YjNCbGNtRjBiM0l0WTJFd0hoY05Nakl3TlRJek1UZ3hNRFE0V2hjTgpNekl3TlRJd01Ua3hNRFE0V2pBZk1SMHdHd1lEVlFRREV4UndiM04wWjNKbGN5MXZjR1Z5WVhSdmNpMWpZVEJaCk1CTUdCeXFHU000OUFnRUdDQ3FHU000OUF3RUhBMElBQlArM2dEb2s3V3duaDZmNnNUV3ozUmlrS3Q4TFhyN0QKSEpGSGNXdHd3MDI5TXQrb0lubWYwUE1VS1BiVHgrSDBSZTBLcENSRUhCbytmcXJqblIzZlBXdWpSVEJETUE0RwpBMVVkRHdFQi93UUVBd0lCQmpBU0JnTlZIUk1CQWY4RUNEQUdBUUgvQWdFQU1CMEdBMVVkRGdRV0JCU3ErUFdhClQreTAvUjBRb1AzUS9nUnZWY3JGQ0RBS0JnZ3Foa2pPUFFRREF3TklBREJGQWlBRHl3UXR2Zk1xUEIvWXlzL1QKd2lNZExNR3JocWVXeDVjYVZ2TWNVWkJxWHdJaEFQS1NBemo5K1RsTzg0cmNFN25pT3U2K2NRWEYzcjNxTFFOYQpNYWVId3d5TAotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
+  tls.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNNVENDQWRlZ0F3SUJBZ0lRUjVzZEwwcit1S0VTUjVudzFvYkFuakFLQmdncWhrak9QUVFEQXpBZk1SMHcKR3dZRFZRUURFeFJ3YjNOMFozSmxjeTF2Y0dWeVlYUnZjaTFqWVRBZUZ3MHlNakExTWpNeE9ERXdORGhhRncweQpNekExTWpNeE9URXdORGhhTURzeE9UQTNCZ05WQkFNVE1IQnliMlIxWTNScGIyNHRjSEpwYldGeWVTNXdjbTlrCmRXTjBhVzl1TG5OMll5NWpiSFZ6ZEdWeUxteHZZMkZzTGpCWk1CTUdCeXFHU000OUFnRUdDQ3FHU000OUF3RUgKQTBJQUJPTWQ3ZUo5U1JtYmd0TUJ1R1hFc1UyNzBOUjhvU0pCSUJEdk1DTGpkSnB0TmVHWCt4MEhHT0ZZRGw0cgpOd0JZUk9EaUc3Z2loaVZ0ZyszTTNhejdjWk9qZ2Rnd2dkVXdEZ1lEVlIwUEFRSC9CQVFEQWdXZ01Bd0dBMVVkCkV3RUIvd1FDTUFBd0h3WURWUjBqQkJnd0ZvQVVxdmoxbWsvc3RQMGRFS0Q5MFA0RWIxWEt4UWd3Z1pNR0ExVWQKRVFTQml6Q0JpSUl3Y0hKdlpIVmpkR2x2Ymkxd2NtbHRZWEo1TG5CeWIyUjFZM1JwYjI0dWMzWmpMbU5zZFhOMApaWEl1Ykc5allXd3VnaUZ3Y205a2RXTjBhVzl1TFhCeWFXMWhjbmt1Y0hKdlpIVmpkR2x2Ymk1emRtT0NIWEJ5CmIyUjFZM1JwYjI0dGNISnBiV0Z5ZVM1d2NtOWtkV04wYVc5dWdoSndjbTlrZFdOMGFXOXVMWEJ5YVcxaGNua3cKQ2dZSUtvWkl6ajBFQXdNRFNBQXdSUUloQVBCai9uVU9HeHpsNXVvQnl6WHNuT3ppbTBJNHFVL21pRS9COFpOcApBSTNNQWlBOWtFeUphUGRqVHZ6cEc4LzZxbldrdGh6K1FKK3h5bGtMenNVUUNld2ZhUT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
+  tls.key: LS0tLS1CRUdJTiBFQyBQUklWQVRFIEtFWS0tLS0tCk1IY0NBUUVFSUlFRlpFS1ZIZ3YyT25uZkljaXlRUFlzbzBvalBVN3NIVS9KUFI3TE5CWERvQW9HQ0NxR1NNNDkKQXdFSG9VUURRZ0FFNHgzdDRuMUpHWnVDMHdHNFpjU3hUYnZRMUh5aElrRWdFTzh3SXVOMG1tMDE0WmY3SFFjWQo0VmdPWGlzM0FGaEU0T0lidUNLR0pXMkQ3Y3pkclB0eGt3PT0KLS0tLS1FTkQgRUMgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
❓ How'd you generate these? When do they expire? 🤔 The answers should probably be written near here, too. (comment, readme, whatever.)

Reply:
I forget how to generate certs using `openssl` or any other tool, so those steps would help me in the future. All certs expire _eventually_; we'll need to regenerate compatible ones then. If it's "far enough" out we can punt. 🤷🏻